back/engines/commercial/analytics/app/services/analytics/populate_dimensions_service.rb

```diff
@@ -23,6 +23,31 @@ def create_dates(from, to)
         end
       end
 
+      def populate_types
+        types = [
+          { name: 'idea', parent: 'post' },
+          { name: 'initiative', parent: 'post' },
+          { name: 'comment', parent: 'initiative' },
+          { name: 'comment', parent: 'idea' },
+          { name: 'reaction', parent: 'initiative' },
+          { name: 'reaction', parent: 'idea' },
+          { name: 'reaction', parent: 'comment' },
+          { name: 'poll', parent: nil },
+          { name: 'volunteer', parent: nil },
+          { name: 'survey', parent: nil },
+          { name: 'basket', parent: nil },
+          { name: 'event_attendance', parent: nil },
+          *Follower::FOLLOWABLE_TYPES.map { |type| { name: 'follower', parent: type.downcase } }
+        ]
+
+        current_types = Analytics::DimensionType.all.as_json(only: %i[name parent])
+
+        return unless current_types & types != types
+
+        Analytics::DimensionType.delete_all
+        Analytics::DimensionType.insert_all(types)
```

Comment:
If we keep that way, we can make it transactional:
```suggestion
        Analytics::DimensionType.transaction do
          Analytics::DimensionType.delete_all
          Analytics::DimensionType.insert_all(types)
        end
```